### DISCUSSION TITLE
GPL License and share

### DISCUSSION THREAD

**Discussion by J-Jamet** [Kunzisoft/KeePassDX#1116] [General]
Recently I opened an issue on the project "https://github.com/authpass/kdbx.dart/issues/4#issuecomment-925284671", because the library under GPL3 license is used for another project whose source code is not accessible.

Small question for the community :
What do you think about the application of licenses, is it something you don't care about or on the contrary that really matter in the world of open source applications?

**Comment by ghost** [comment]
I don't particularly care as long as no one is trashing app/library authors choice of project licence just because they don't like it and should respect it.

**Reply by J-Jamet** [reply]
I agree, it's all about respect in the first place. Using other people's work and profiting from it and even denigrating it afterwards is not what I call respect.
